ssl/record/methods/tls_common.c

```diff
@@ -863,6 +863,11 @@ int tls_get_more_records(OSSL_RECORD_LAYER *rl)
                 enc_err = 0;
             if (thisrr->length > SSL3_RT_MAX_COMPRESSED_LENGTH + mac_size)
                 enc_err = 0;
+#ifdef FUZZING_BUILD_MODE_UNSAFE_FOR_PRODUCTION
+            if (enc_err == 0 && mac_size > 0 && (md[0] ^ thismb->mac[0]) != 0xFF) {
```

Comment:
I try to have the fuzzer still get errors, but only if every bit of the first byte is wrong ie `(md[0] ^ thismb->mac[0]) == 0xFF`

Reply:
Any reason you pick 0xFF and not say 0x0? It doesn't really matter for me.